repl/src/main/scala/org/apache/spark/repl/ExecutorClassLoader.scala

```diff
@@ -54,7 +54,7 @@ class ExecutorClassLoader(
     classUri: String,
     parent: ClassLoader,
     userClassPathFirst: Boolean) extends ClassLoader(null) with Logging {
-  val uri = new URI(classUri)
+  val uri = new URI(classUri.replace('\\', '/'))
```

Comment:
Hi, @guolianwei . We use `Utils.isWindows` to use Window-specific code.

https://github.com/apache/spark/blob/b6aea1a8d99b3d99e91f7f195b23169d3d61b6a7/launcher/src/main/java/org/apache/spark/launcher/Main.java#L90-L91